csearch.py

```diff
@@ -73,7 +75,7 @@ def _on_search(self, result):
                             view=view, erase=True)
         view.set_status('YetAnotherCodeSearch', 'Searching...')
         try:
-            s = settings.get_project_settings(self.window.project_data())
+            s = settings.get_project_settings(self.window.project_data(), self.window.project_file_name())
```

Comment:
Line is longer than 79 characters, as per PEP8 - https://www.python.org/dev/peps/pep-0008. Care to validate PEP8 compliance? You can use the pep8 tool or http://pep8online.com/.